.github/workflows/style.yml

```
@@ -0,0 +1,18 @@
+name: Style
+
+on: [push, pull_request]
```

It's probably sufficient just to run this workflow on `push`. I don't think it needs to be re-run when a PR is opened.
```suggestion
on: [push]
```